src/server/app.js

```diff
@@ -1,13 +1,36 @@
-var express = require('express');
-var app = express();
+/* 
 
+	app.js
+
+	Initializes server. Sets up static content. Establishes routes and 
+	their handlers.
+
+	Written by Sam Reaves
+	July 22, 2015
+
+ */
```

Comment:
Point is, try out WebStorm, watch some youtube tutorials on some of its features,and be amazed. And if you like it we can get you a license.